frontend/taipy-gui/dom/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "4.0.0",
   "private": true,
   "dependencies": {
-    "react": "^18.2.0",
+    "react": "^18.3.1",
```

Comment:
looks like snyk doesn't understand semantic version management
react is 18.2.0 in package.json
and 18.3.1 in package-lock.json (which indicates the installed version